component/service/src/main/java/org/exoplatform/social/rest/impl/user/UserRestResourcesV1.java

```diff
@@ -1321,7 +1321,8 @@ private String importUser(UserImportResultEntity userImportResultEntity,
     boolean onboardUser = !userObject.isNull("onboardUser") && userObject.getString("onboardUser").equals("true");
 
     User existingUser = organizationService.getUserHandler().findUserByName(userName, UserStatus.ANY);
-    if (existingUser != null) {
+    if (existingUser != null && !userObject.isNull("password")) {
```

Comment:
IMO we could log inndebug mode here to tell when we skipped password  writing . it may be useful to separate updates from initial rights or for feneral troubleshooting